pkg/controller/plan/adapter/ova/builder.go

```diff
@@ -440,6 +440,12 @@ func (r *Builder) Tasks(vmRef ref.Ref) (list []*plan.Task, err error) {
 	return
 }
 
+func (r *Builder) InstanceTypeName(vmRef ref.Ref) (name string, err error) {
+	// The VM is build from configuration, we don't need the instancetype
```

Comment:
not sure I get this part - why don't we need the instance type? (and specifically, when we otherwise we set the template)

Reply:
ok I see, so it's not because it's built from configuration but since we do not take the operating system within the OVF into account, we can do it like we do in oVirt. CC @bkhizgiy